docs/user-guide.md

```diff
@@ -779,25 +793,22 @@ Most parameters have default values, so you only need to specify them when you
 need a different value. The parameter values that the toolkit uses are echoed
 during execution so you can confirm the configuration.
 
-The complete list of parameters and their values are provided in the [Parameters
-section](#heading=h.m1e7sxtn4kzm).
-
 Configuring an installation is discussed in [Configuring
 installations](#configuring-installations).
```

Comment:
I was suggesting that you delete the sentence "Configuring an installation is discussed in [Configuring 
installations](#configuring-installations)."

I think the sentence about the list of parameters is okay to leave as-is

Reply:
Done.